modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -258,6 +261,53 @@ export default {
     async create() {
       await this.edit(null);
     },
+    async handleUtilityOperation(action) {
+      const operation = this.utilityOperations.menu
+        .find((op) => op.action === action);
+
+      if (!operation) {
+        return;
+      }
+
+      const {
+        modal, body, ...modalOptions
+      } = operation.modalOptions || {};
+
+      if (modal) {
+        await apos.modal.execute(modal, {
+          action: this.moduleOptions.action,
+          route: operation.route,
```

Comment:
We need to decide if we're going to support the route option here before this gets published. We removed it for batch operations.

Reply:
I would suggest we remove it and use the action. We can always add this later for more flexibility.

Reply:
I agree. Remove it for now. Later it can be the default value of the option.